MapboxNavigation/NavigationViewController.swift

```diff
@@ -8,195 +8,10 @@ import CarPlay
 #endif
 
 /**
- The `NavigationViewControllerDelegate` protocol provides methods for configuring the map view shown by a `NavigationViewController` and responding to the cancellation of a navigation session.
+ A ContainerViewController is any UIViewController that subscribes to the NavigationComponent messaging protocol.
```

Comment:
A class _subscribes_ to a notification but _conforms_ to a protocol, even if the protocol is used for messaging of some sort.

```suggestion
 A ContainerViewController is any UIViewController that conforms to the NavigationComponent messaging protocol.
```